src/components/Select2.js

```diff
@@ -2,6 +2,7 @@ import React, { Component, PropTypes } from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import 'select2';
+import shallowEqual from "react/node_modules/fbjs/lib/shallowEqual";
```

Comment:
shallowEqual is internal API for react. There are 2 ways to use it. 
1. get it from react, `+` - we haven't any new dependencies `-` - it's tricky
2. install fbjs `+` - it's normal usage `-` - on my computer npm installl this lib separately for react and for react-select2-wrapper